Bigtable/tests/Unit/RowMutationTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * Copyright 2018, Google LLC All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Bigtable\Tests\Unit;
+
+use Google\Cloud\Bigtable\RowMutation;
+use Google\Cloud\Bigtable\V2\MutateRowsRequest_Entry;
+use Google\Cloud\Bigtable\V2\Mutation;
+use Google\Cloud\Bigtable\V2\Mutation_DeleteFromColumn;
+use Google\Cloud\Bigtable\V2\Mutation_DeleteFromFamily;
+use Google\Cloud\Bigtable\V2\Mutation_DeleteFromRow;
+use Google\Cloud\Bigtable\V2\Mutation_SetCell;
+use Google\Cloud\Bigtable\V2\TimestampRange;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @group bigtable
+ * @group bigtabledata
+ */
+class RowMutationTest extends TestCase
+{
+    const ROW_KEY = 'r1';
+    const COLUMN_FAMILY = 'cf1';
+    const COLUMN_QUALIFIER = 'cq1';
+    const VALUE = 'value1';
+
+    private $rowMutation;
+
+    public function setUp()
+    {
+        $this->rowMutation = new RowMutation(self::ROW_KEY);
+    }
+
+    public function testGetRowKey()
+    {
+        $this->assertEquals(self::ROW_KEY, $this->rowMutation->getRowKey());
+    }
+
+    public function testUpsertWithTimeRange()
+    {
+        $this->rowMutation->upsert(self::COLUMN_FAMILY, self::COLUMN_QUALIFIER, self::VALUE, 20);
+
+        $entry = $this->rowMutation->getEntry();
+        $mutationSetCell = new Mutation_SetCell;
+        $mutationSetCell->setFamilyName(self::COLUMN_FAMILY)->setColumnQualifier(self::COLUMN_QUALIFIER)
+                        ->setValue(self::VALUE)
```

Comment:
May worth checking with @jdpedrie @dwsupplee.
Do we usually indent for 4 spaces in this case?

Reply:
Yep. If you're splitting this onto multiple lines, though, split on each chained method:

```php
        $mutationSetCell->setFamilyName(self::COLUMN_FAMILY)
            ->setColumnQualifier(self::COLUMN_QUALIFIER)
            ->setValue(self::VALUE)
            ->setTimestampMicros(20);
```

Reply:
done